packages/blockchain-extension/tutorials/new-tutorials/basic-tutorials/a10.md

```diff
@@ -1,6 +1,6 @@
 <!-- Standard Header Start -->
 **IBM Blockchain Platform**
-<h3 align='left'>← <a href='./a9.md'><b>A9: Publishing an event</b></a>
+<h3 align='left'></h3> <a href='./a9.md'><b>A9: Publishing an event</b></a>
```

Comment:
You've removed the arrow in all these instances :scream:

Reply:
Ah snap, will fix that now! 😓